scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -1661,19 +1692,14 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
 
   def caseClause(): Case = atPos(in.prevTokenPos, auto) {
     require(token.isNot[`case`] && debug(token))
-    token match {
-      case token: Unquote if token.prev.is[Ellipsis] =>
-        unquote[Case]
```

Comment:
Would it be possible to retain this pattern match clause in `caseClause` (more precisely, changing it to `case token: Unquote if token.prev.is[Case]`) and then migrating `caseClauses` back to `caseSeparated`?